src/pages/_error.tsx

```diff
@@ -20,27 +23,40 @@ const Error: NextPageWithLayout<Props> = ({ statusCode }: Props) => {
 
   return (
     <>
+      <h1>500</h1>
       <GridContainer>
         <section>
           <Logo noText />
 
           <h2>Houston, we have a problem</h2>
 
           <h3>
-            That’s {statusCode ? `an internal server error` : 'a client error'}.
-            While we work on fixing that, let’s get you back to business.
+            That’s an internal server error. While we work on fixing that, let’s
+            get you back to business. You may also submit a report to us at{' '}
+            <a
+              href={`mailto:${FEEDBACK_EMAIL}?subject=${FEEDBACK_SUBJECT}`}
+              target="_blank"
+              rel="noreferrer noopener"
+              className="usa-link">
+              {FEEDBACK_EMAIL}
+            </a>{' '}
```

Comment:
Question: This is not a pattern I'm familiar with, is it to ensure a space in the rendered page? Would it be better as an `&nbsp;` or is that different? I'm fine with it either way.

Reply:
I copied it over from the feedback card component, i do not have strong opinions about it